task_spec/spec/acceptance/init_spec.rb

```diff
@@ -46,6 +46,7 @@ def latest_platform_list
       el-9-aarch64|
       ubuntu-22\.04-aarch64|
       amazon-2023|
+      amazon-2-aarch64|
```

Comment:
does that add support for amazon-2 only on aarch64 or also for AMD? then you could maybe use `amazon-2-`?